src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs

```diff
@@ -137,19 +138,27 @@ private static void ReinitializeAppSettings()
             }
         }
 
-        private static ScriptHostConfiguration GetScriptHostConfiguration(string scriptPath, string logPath)
```

Comment:
Can you explain this change? Looks like we had an overload previously that was only used by tests, is that is?